src/lib/webext.ts

```diff
@@ -153,3 +154,74 @@ export async function openInNewTab(
 export async function openInNewWindow(createData = {}) {
     browserBg.windows.create(createData)
 }
+
+export async function openInTab(tab, strarr: string[]) {
+    let address = strarr.join(" ")
+
+    if (address == "") {
+        address = config.get("newtab")
+    }
+
+    const index = address.indexOf(" ")
+    let firstWord = address
+    if (index > -1)
+        firstWord = address.substr(0, index)
+
+    if (firstWord == "") {
+        // No query, no newtab set, the user is asking for Tridactyl's newtab page
+        return browserBg.tabs.update(tab.id, {url: "/static/newtab.html"})
+    }
+
+    // Perhaps the user typed an URL?
```

Comment:
FWIW - "a URL" is more correct than "an URL" because, when spoken, "URL" doesn't sound like it starts with a vowel.